sklearn/datasets/tests/test_svmlight_format.py

```diff
@@ -139,19 +148,23 @@ def test_load_compressed():
     assert_array_almost_equal(y, ybz)
 
 
+@fails_if_pypy
```

Comment:
No way to do a module-level xfail rather than mark each test?

Reply:
It should be possible using `pytestmark`: https://docs.pytest.org/en/latest/example/markers.html#marking-whole-classes-or-modules